### ISSUE TITLE
Replace bash scripts with a single python CLI tool

### ISSUE THREAD

**Issue by cgmcintyr** [SoftwareQuTech/SimulaQron#100]
## Problem
Currently running a SimulaQron network involves a number of bash scripts:

```
run/startAll.sh
run/startAllLog.sh
run/startCQCNodes.sh
run/startVNodes.sh
```

These scripts do not have a standard interface (e.g. "--help" prints help message) and are difficult to understand for people without knowledge of Linux and bash scripting. 

Furthermore bash has a large number of pit-falls which are easy to fall into (e.g. forgetting to escape variables using "" can cause [problems](https://www.tldp.org/LDP/abs/html/quotingvar.html)).

## Proposed Solution
Moving to a single python CLI entry-point for interacting with a SimulaQron network.

## Benefits
- Cross platform support.
- Writing python regression tests for the CLI.
- Merging new features (e.g. #86) into the CLI rather than creating Yet Another Script.
- Easily add new flags/arguments/sub-commands without having to understand bash.

**Comment by cgmcintyr** [comment]
See discussion #96

**Comment by AckslD** [comment]
Fixed in #150
